### DISCUSSION TITLE
Feedback for course "Git, GitHub and Markdown"

### DISCUSSION THREAD

**Discussion by ludelafo** [heig-vd-dai-course/heig-vd-dai-course#2] [Polls]
Can you let us know what was easy and what was difficult for you during this chapter?

This will help us to improve the course and adapt the content to your needs. If we notice some difficulties, we will come back to you to help you.

You can use reactions to express your opinion on a comment! 🚀

Vous pouvez aussi nous laisser votre avis en français, il n'y a pas de souci !

**Comment by MolinaElGringo** [comment]
The markdown is pretty easy once you try to learn it. I liked it.
The git part need a bit more explanation, mostly on the signing with the ssh key in my opinion. 
Using the commands isn't that hard, but when it comes to check the conflict, I'm not very used to it. maybe by doing it a bit more frequently, it will allow people like me to know how to deal with it.

**Reply by ludelafo** [reply]
Thank you for your feedback!

I agree the signing part using Git and SSH is a bit complex at first.

My main issue is how to make it comprehensive and available for all OSes. This is why the easiest way was to redirect you to the official GitHub documentation as it can be maintained more frequently than I could ever do. You can always go back to it for other projects.

Regarding the conflict part, I'm pretty sure you'll be able to experiment this while doing the practical work with your peer. However, if you need to train more, you can always re-follow the practical content to intentionally create more conflicts until you feel comfortable with them.

Feel free to reach us if you need any help :)

**Comment by ludelafo** [comment]
All previous comments have been marked as hidden (outdated) for the start of the new DAI semester.
